src/client/Microsoft.Identity.Client/ManagedIdentity/ServiceFabricManagedIdentitySource.cs

```diff
@@ -45,8 +45,35 @@ public static AbstractManagedIdentity Create(RequestContext requestContext)
             return new ServiceFabricManagedIdentitySource(requestContext, endpointUri, EnvironmentVariables.IdentityHeader);
         }
 
+        internal override HttpClient CreateCustomHttpClient(RequestContext requestContext)
+        {
+            return new HttpClient(CreateHandlerWithSslValidation(requestContext.Logger));
```

Comment:
I think you started looking at reusing this HttpClient with the Lazy<T> version, but didn't wrap it up.